package.json

```diff
@@ -23,6 +23,7 @@
     "grunt-contrib-jshint": "0.11.0",
     "grunt-jscs": "1.5.0",
     "grunt-mocha-cli": "1.12.0",
+    "istanbul": "^0.4.3",
```

Comment:
still here!  should scrap it.  don't you go on blamin' poor ol' c-v for being a hard workin' soul...